Fix: The prometheus-monitoring pod is restarting (#4633)
kyma-project/kyma: docs/monitoring/05-03-prometheus.md

| Parameter | Description | Default value |
|-----------|-------------|---------------|

> | **retention** | Specifies a period of time for which Prometheus stores the metrics in-memory. This retention time is for the in-memory storage only, Prometheus will keep recent data for configured time in-memory to avoid read whole data from disk.| `2h` |

Reviewer: ```suggestion
| **retention** | Specifies a period of time for which Prometheus stores the metrics in-memory. This retention time applies to in-memory storage only. Prometheus keeps the recent data for the specified time in-memory to avoid reading the entire data from disk.| `2h` |
```

> | **retention** | Specifies a period of time for which Prometheus stores the metrics in-memory. This retention time applies to in-memory storage only. Prometheus keeps the recent data for the specified time in-memory to avoid reading the entire data from disk.| `2h` |

Reviewer: ```suggestion
| **retention** | Specifies a period of time for which Prometheus stores the metrics in-memory. This retention time applies to in-memory storage only. Prometheus stores the recent data in-memory for the specified amount of time to avoid reading the entire data from disk.| `2h` |
```
